docs/upgrade.md

```diff
@@ -32,14 +25,13 @@ spec:
 
 ### Managing available versions
 
-An upgrade channel file (
-`rancheros-v0.0.0-amd64.upgradechannel-amd64.yaml` ) file is shipped
-in Elemental releases and can be applied in a Kubernetes cluster where the rancheros operator is installed to syncronize available version for upgrades.
+An upgrade channel file can be applied in a Kubernetes cluster where the elemental operator is installed to syncronize available version for upgrades.
 
 
-For instance an upgrade channel file might look like this and is sufficient to `kubectl apply` it where the ros-operator is installed: 
+For instance an upgrade channel file might look like this and is sufficient to `kubectl apply` it to the Rancher management cluster:
+
 ```yaml
-apiVersion: rancheros.cattle.io/v1
+apiVersion: elemental.cattle.io/v1
 kind: ManagedOSVersionChannel
 metadata:
   name: os2-amd64
```

Comment:
os2 name and below the REPOSITORY and IMAGE_PREFIX point to real os2 repos, I would change them for made up repos